exasol/bucketfs/__init__.py

```diff
@@ -205,7 +235,7 @@ def __iter__(self) -> Iterator[str]:
         yield from self.files
 
     def upload(
-        self, path: str, data: Union[ByteString, BinaryIO, Iterable[ByteString]]
+        self, path: str, data: ByteString | BinaryIO | Iterable[ByteString]
```

Comment:
:+1:  Cool syntax!

Reply:
which doesn't work for python 3.8

@Nicoretti you need to wait with this a bit longer

Reply:
@tkilias it's fine check the ci `from __future__ import annotations` does the trick ;)

Reply:
to be more precise, [PEP 563 – Postponed Evaluation of Annotations](https://peps.python.org/pep-0563/) does the trick.

Reply:
ok, that its fine